### DISCUSSION TITLE
계정 팔로우

### DISCUSSION THREAD

**Discussion by limitless** [naverz/zepeto-studio-kor#948] [5. Q&A]
안녕하세요

사용자나 특정 포인트에 접근하여 계정 팔로우 할수 있도록하는데오
가능할까요?

감사합니다.

**Comment by limitless** [comment]
<img width="1504" alt="스크린샷 2022-07-04 오전 11 43 26" src="https://user-images.githubusercontent.com/63586725/177072446-f4d1f88a-3283-4685-9107-efa9a9200f46.png">
참고로 스크린샷 올립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 플레이어를 터치하면 미니 프로필이 뜨도록 기본 설정이 되어 있습니다.
ZEPETO 캐릭터 컨트롤 가이드 하단의 FloatingUI(onApp) 부분을 참고하여 주세요. 
https://studio.zepeto.me/kr/guides/character-control

다만 미니 프로필을 특정 상황에서 팝업 시키는 API는 현재 제공하고 있지 않습니다.
감사합니다.
